MyTwitter.psm1

```diff
@@ -157,83 +159,58 @@ Function New-MyTwitterConfiguration {
 		[switch]$Force
 	)
 	begin {
-		$RegKey = 'HKCU:\Software\MyTwitter'
+		$JSONPath = '.\MyTwitter.json'

```

Comment:
would it be more sensible to default to a more specific path (either from `$PSScriptRoot` or `$env:HOME`) - `~/.mytwitter/` for example. Also, while pwsh will parse out slashes in filenames according to the OS, it's good practice to use `/` when coding for core.

Reply:
Thanks for your input.  This is just a first draft regarding issue #20. Sorry, if I've been a bit hasty in creating a PR.

I agree that the current behavior regarding path is not ideal. 
imo there are two options:
- Change Cmdlets so that the path to the JSON can/must be manually provided.
- Find a behavior that is usable on Windows, Linux & Mac and in Windows Powershell and Core. `$PSScriptRoot` could be a good solution.

Also, thanks for the Slash suggestion. Its a bad habit I need to change and will do so. ;)